tensorflow/lite/kernels/cpu_backend_context.cc

```diff
@@ -19,6 +19,9 @@ limitations under the License.
 #include "tensorflow/lite/experimental/ruy/context.h"
 
 namespace tflite {
+// For legacy reasons, we use 4 threads by default unless the thread count is
+// explicitly specified by the context.
+const int kDefaultNumThreads = 4;
```

Comment:
We have to discuss this internally. In legacy code it was 4 for float, 1 for quantized. Stay tuned.